tools/docker-compose/Dockerfile

```diff
@@ -121,9 +121,9 @@ ENV LC_ALL en_US.UTF-8
 
 ADD tools/docker-compose/nginx.conf /etc/nginx/nginx.conf
 ADD tools/docker-compose/nginx.vh.default.conf /etc/nginx/conf.d/nginx.vh.default.conf
-ADD tools/docker-compose/start_development.sh /start_development.sh
+ADD tools/docker-compose/launch_awx.sh /launch_awx.sh
```

Comment:
Where is `/launch_awx.sh` getting called? Do you need to update the file `tools/docker-compose.yml`?

Reply:
:man_facepalming: yep.  Updated